File: uservice-reports/main/src/test/scala/sinnet/MinutesTests.scala
package sinnet

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test
import sinnet.reports.Minutes

class MinutesTests {

    @Test
    def shouldFormatString: Unit = {

      Assertions.assertThat(Minutes(15).asString).isEqualTo("0:15");

      Assertions.assertThat(Minutes(7381).asString).isEqualTo("123:01");

    }
}

File: uservice-timeentries/main/host/src/test/java/sinnet/reports/MinutesTests.java
package sinnet.reports;

import org.assertj.core.api.Assertions;
import org.junit.jupiter.api.Test;

import sinnet.reports.shared.Minutes;

class MinutesTests {

    @Test
    void shouldFormatString() {

      Assertions.assertThat(Minutes.of(15).asString()).isEqualTo("0:15");

      Assertions.assertThat(Minutes.of(7381).asString()).isEqualTo("123:01");

    }
}
